RetailSuiteWarehousing/Overview/02_Features.md

```diff
@@ -0,0 +1,29 @@
+# Key features
+
+## Integrated warehouse management solution for efficient e-commerce
+
+- Complete warehouse management
+    Actindo *Warehousing* module allows you to centrally control and manage material flow and stock levels. It provides a complete overview of all materials and warehouses, as well as updating your stocks in real time. Furthermore, the management is possible down to the individual storage shelf.
+
+- Support of different warehouse types  
+    Whether it is a picking warehouse, a replenishment warehouse, or a transit warehouse, you can use the *Actindo Core1 Platform* to manage all of your company's warehouses. Different warehouse types are supported, so you can always have everything at a quick glance. 
+
+- Stock allocation matrix
+    With the help of our unique specially developed stock allocation matrix, stocks can be divided and displayed separately for each of your different sales channels. This enables flexible stock allocation for your different sales channels and according to your specific requirements. 
+
+- Stock withdrawal matrix  
+    This function allows you to automate and optimize the storage location selection, that is, from which warehouse and under which conditions stock is withdrawn. You can customize it easily, using parameters such as delivery country, sales channel or shipping provider.
+
+- Posting history
+    You can track the entire posting history for all incoming and outgoing materials in our Actindo *Warehousing* module. This way you can always keep track of any stock movements in real time. 
+
+[comment]: <> (IQS-Möglichkeit erwähnen, auch wenn Doku nicht beschrieben?)
+
+## Central control from order input to return
+
+**Simple shipping process controlled from a single source** 
+
+Well-organized and manageable warehouse management is the basis for successful e-commerce. The better the various links in your supply chain, from purchasing through warehousing to returns, are technically interconnected, the more efficient the management of all warehouse and logistics processes is.
+
```

Comment:
Satz ist ein bisschen verwirrend
Vorschlag
The better the various links in your supply chain are technically interconnected, from purchasing through warehousing to returns, the more efficient the management of all warehouse and logistics processes is.